src/style.css

```
@@ -0,0 +1,123 @@
+body {
+  font-family: "Avenir-Book", sans-serif;
+  font-size: 16px;
+  margin: 8px;
+  display: block;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search-bar {
+  width: 100%;
+  margin: 20px auto 0;
+  display: flex;
+  justify-content: center;
+}
+
+.search-bar__large {
+  height: 70px;
+}
+
+.input-label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.input-label__large::after {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.input-label::after {
+  background: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  outline: none;
+}
+
+.input__large {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+::placeholder {
+  font-family: "Avenir-Book", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.search-bar:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 3px;
+}
+
+.input:focus {
+  outline: none;
+  font-family: "Avenir-Heavy", sans-serif;
+  font-weight: 600;
+}
```

this is where you need to apply `linear-gradient`
currently it's not applied at all - give a closer look on the deployed page.
```suggestion
.input:focus {
  // outline: none; // not sure what this style does. Seems redundant
  font-family: "Avenir-Heavy", sans-serif;
  font-weight: 600;
  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
}
```

---

Okay noted thanks have fixed it.